pwa/src/pages/login/index.tsx

```diff
@@ -100,7 +100,8 @@ const Login: React.FC = () => {
       />
 
       <div className="login-container">
-        <h1>Welcome to Conductor!</h1>
+        <h1>Welcome to the</h1>
+        <h1>Common Gateway!</h1>
```

Comment:
This is semantically incorrect; a page cannot have two `h1` tags.

I expect you've added the two separate tags to create some layout. I'd recommend doing that with `css`. 

E.g.:

```
<h1 className="example">Welcome to the Common Gateway!</h1>

...

.example {
  max-width: ...;
}
```